Pipfile.lock

```diff
@@ -18,12 +18,12 @@
     "default": {
         "apluggy": {
             "hashes": [
-                "sha256:ecfcd7cad1c484281cd528281104c53657297f8ab8e723f26c2f396a407c8d0c",
-                "sha256:fe66316ce1cdd9d7c406bd39582c267df5d6cf70b701bdbe9523d5efecd45308"
+                "sha256:9261e12d60720c4f92189cd7f923e1de1e7aa536cd22e9c5770c289389d5609c",
+                "sha256:985bea17ab2f4062e5d6f5094ea025376ac4f255bf0e7c6b2aadf467ae2ee4da"
             ],
             "index": "pypi",
-            "markers": "python_version >= '3.8'",
-            "version": "==0.9.4"
+            "markers": "python_version >= '3.10'",
```

Comment:
this is definitely unwanted

Reply:
maybe from this?
- https://github.com/simonsobs/apluggy/commit/df266b9b382a7105c8eac823a0034016e43372a9#commitcomment-141598247

Reply:
seems it came from here:
- https://github.com/simonsobs/apluggy/commit/cdd0f16b187d9e4cf7f578292c9a40a774d80113